lib/phx_tools_web/live/phx_tools_live/instruction_components.ex

```diff
@@ -3,24 +3,28 @@ defmodule PhxToolsWeb.InstructionComponents do
   Shows the steps to execute different scripts depending on the operating system.
   """
   use PhxToolsWeb, :html
+  use Phoenix.Component
```

Comment:
It's already used:
https://github.com/nittin-shankar/phx.tools/blob/040e79747f77e5e2b7bffd0a34d0db26c60d920f/lib/phx_tools_web.ex#L73